app/src/main/java/com/funcheap/funmapsf/commons/utils/DateCostFormatter.java

```diff
@@ -28,8 +25,7 @@
     public static String formatDate(String date)  {
         String retDate = date;
         try {
-            SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss", Locale.ENGLISH);
-            sdf.setTimeZone(TimeZone.getTimeZone("PDT"));
```

Comment:
@ganeshskudva Please do not set the time zone for your SimpleDateFormatters. It causes all our times to be offset by 8 hours. You only need to use this when we are receiving time in GMT and we need to offset it ourselves.